tests/cli/test_import_schema.py

```diff
@@ -0,0 +1,490 @@
+    output = sdl_importer.import_sdl(schema)
+    assert output
+
+
+# Complex object
+def test_import_specific_object_type(mocker):
+    s = '''
+    """A single film."""
+    type Film implements Node {
+    """The title of this film."""
+    title: String
+
+    """The episode number of this film."""
+    episodeID: Int
+
+    """The opening paragraphs at the beginning of this film."""
+    openingCrawl: String
+
+    """The name of the director of this film."""
+    director: String
+
+    """The name(s) of the producer(s) of this film."""
+    producers: [String]
+
+    """The ISO 8601 date format of film release at original creator country."""
+    releaseDate: String
+    speciesConnection(after: String, first: Int, before: String, last: Int):
+    FilmSpeciesConnection
+    starshipConnection(after: String, first: Int, before: String, last: Int):
+    FilmStarshipsConnection
+    vehicleConnection(after: String, first: Int, before: String, last: Int):
+    FilmVehiclesConnection
+    characterConnection(after: String, first: Int, before: String, last: Int):
+    FilmCharactersConnection
+    planetConnection(after: String, first: Int, before: String, last: Int):
+    FilmPlanetsConnection
+
+    """The ISO 8601 date format of the time that this resource was created."""
+    created: String
+
+    """The ISO 8601 date format of the time that this resource was edited."""
+    edited: String
+
+    """The ID of an object"""
+    id: ID!
+    }
+    '''
+    output = sdl_importer.import_sdl(s)
+    assert output
+
+
+# region Scalars
+# Boolean
+def test_import_bool_field():
+    """
+    Test for a required Boolean field type
+    with a field description and case change
+    """
+    s = '''
+    type Woman {
+        """
+        If a woman weighs less than a duck,
+        then she is a...?
+        """
+        isWitch: Boolean!
+    }
+    '''
+    output = sdl_importer.import_sdl(s)
+    what_it_should_be = (
+        "import strawberry\n"
+        "\n"
+        "\n"
+        "@strawberry.type\n"
+        "class Woman:\n"
+        "    is_witch: bool = strawberry.field(\n"
+        "        name='isWitch',\n"
+        "        description='''If a woman weighs less than a duck,\n"
+        "then she is a...?'''\n"
+        "    )"
+    )
+
+    assert output == what_it_should_be
+
+
+def test_import_optional_bool_field():
+    """
+    Test for a optional Boolean field type
+    with a field description and case change
+    """
+    s = '''
+    type Woman {
+        """
+        If a woman weighs less than a duck,
+        then she is a...?
+        """
+        isWitch: Boolean
+    }
+    '''
+    output = sdl_importer.import_sdl(s)
+    what_it_should_be = (
+        "import typing\n\n"
+        "import strawberry\n"
+        "\n"
+        "\n"
+        "@strawberry.type\n"
+        "class Woman:\n"
+        "    is_witch: typing.Optional[bool] = strawberry.field(\n"
+        "        name='isWitch',\n"
```

Comment:
do you can we can remove name='abc' when the name is the same as `to_camel_case(field_name)`? :)

Reply:
I think it already works that way, will check though.

Reply:
I checked it, if you're referring to leaving out the name attribute from strawberry.field in the case where field name is the same in both camel and snake variant, it already works that way. There are also some examples in tests.